

Starting Toontown Online...
Current time: Thu Jan  5 23:24:27 2023 Eastern Standard Time
sys.path =  ['', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\python27.zip', 'C:\\Panda3D-1.11.0-x64', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\DLLs', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\lib', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\lib\\plat-win', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\lib\\lib-tk', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\bin', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\lib\\site-packages']
sys.argv =  ['C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\toontown\\launcher\\TTOffQuickStartLauncher.py']
generating standard configrc
os.environ =  {'TMP': 'C:\\Users\\theph\\AppData\\Local\\Temp', 'COMPUTERNAME': 'DESKTOP-7054VC9', 'PROCESSOR_LEVEL': '6', 'USERDOMAIN': 'DESKTOP-7054VC9', 'NEKO_INSTPATH': 'C:\\HaxeToolkit\\neko', 'INTEL_DEV_REDIST': 'C:\\Program Files (x86)\\Common Files\\Intel\\Shared Libraries\\', 'PSMODULEPATH': 'C:\\Program Files\\WindowsPowerShell\\Modules;C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules', 'COMMONPROGRAMFILES': 'C:\\Program Files\\Common Files', 'PROCESSOR_IDENTIFIER': 'Intel64 Family 6 Model 158 Stepping 13, GenuineIntel', 'PROGRAMFILES': 'C:\\Program Files', 'PROCESSOR_REVISION': '9e0d', 'PATH': 'C:\\Panda3D-1.11.0-x64\\panda3d\\..\\bin;C:\\Program Files (x86)\\Common Files\\Intel\\Shared Libraries\\redist\\intel64_win\\compiler;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\Program Files\\Python38\\Scripts\\;C:\\Program Files\\Python38\\;C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;C:\\WINDOWS\\System32\\OpenSSH\\;C:\\Program Files\\NVIDIA Corporation\\NVIDIA NvDLISR;C:\\Program Files\\dotnet\\;C:\\Users\\theph\\AppData\\Local\\Programs\\Python\\Python39;C:\\Users\\theph\\AppData\\Local\\Programs\\Python\\Python39\\Scripts;C:\\Program Files\\Common Files\\Autodesk Shared\\;C:\\Program Files\\Git\\cmd;C:\\Program Files (x86)\\Windows Live\\Shared;C:\\HaxeToolkit\\haxe;C:\\HaxeToolkit\\neko;C:\\Program Files\\Microsoft SQL Server\\150\\Tools\\Binn\\;C:\\Program Files\\Microsoft SQL Server\\Client SDK\\ODBC\\170\\Tools\\Binn\\;C:\\Users\\theph\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Users\\theph\\.dotnet\\tools;C:\\Program Files\\JetBrains\\PyCharm Community Edition 2020.3.3\\bin;C:\\Users\\theph\\AppData\\Local\\atom\\bin;C:\\Users\\theph\\AppData\\Local\\GitHubDesktop\\bin;C:\\Panda3D-1.9.0-x64\\python;C:\\Panda3D-1.9.0-x64\\python\\Scripts;C:\\Panda3D-1.9.0-x64\\bin;C:\\Panda3D-1.9.0\\python;C:\\Panda3D-1.9.0\\python\\Scripts;C:\\Panda3D-1.9.0\\bin;C:\\Panda3D-1.10.0\\python;C:\\Panda3D-1.10.0\\python\\Scripts;C:\\Panda3D-1.10.0\\bin;C:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.2.2\\bin;;C:\\Users\\theph\\AppData\\Local\\Programs\\Microsoft VS Code\\bin;C:\\Panda3D-1.11.0\\python;C:\\Panda3D-1.11.0\\python\\Scripts;C:\\Panda3D-1.11.0\\bin;C:\\Users\\theph\\.dotnet\\tools;C:\\Panda3D-1.10.11-x64\\python;C:\\Panda3D-1.10.11-x64\\python\\Scripts;C:\\Panda3D-1.10.11-x64\\bin;C:\\Panda3D-1.11.0-x64\\python;C:\\Panda3D-1.11.0-x64\\python\\Scripts;C:\\Panda3D-1.11.0-x64\\bin;C:\\EventHorizon-battle-ng-teaser\\dependencies\\panda\\python;C:\\EventHorizon-battle-ng-teaser\\dependencies\\panda\\python\\Scripts;C:\\EventHorizon-battle-ng-teaser\\depe', 'SYSTEMROOT': 'C:\\WINDOWS', 'PROGRAMFILES(X86)': 'C:\\Program Files (x86)', 'COMSPEC': 'C:\\WINDOWS\\system32\\cmd.exe', 'DRIVERDATA': 'C:\\Windows\\System32\\Drivers\\DriverData', 'PPYTHON_PATH': '"Panda3D-1.11.0-x64\\python\\ppython.exe"', 'TEMP': 'C:\\Users\\theph\\AppData\\Local\\Temp', 'FPS_BROWSER_APP_PROFILE_STRING': 'Internet Explorer', 'COMMONPROGRAMFILES(X86)': 'C:\\Program Files (x86)\\Common Files', 'PROCESSOR_ARCHITECTURE': 'AMD64', 'ALLUSERSPROFILE': 'C:\\ProgramData', 'USERPROFILE': 'C:\\Users\\theph', 'LOCALAPPDATA': 'C:\\Users\\theph\\AppData\\Local', 'FPS_BROWSER_USER_PROFILE_STRING': 'Default', 'HOMEPATH': '\\Users\\theph', 'USERDOMAIN_ROAMINGPROFILE': 'DESKTOP-7054VC9', 'PROGRAMW6432': 'C:\\Program Files', 'USERNAME': 'theph', 'LOGONSERVER': '\\\\DESKTOP-7054VC9', 'PROMPT': '$P$G', 'SESSIONNAME': 'Console', 'PROGRAMDATA': 'C:\\ProgramData', 'ONEDRIVE': 'C:\\Users\\theph\\OneDrive', 'ADSK_3DSMAX_X64_2019': 'C:\\Program Files\\Autodesk\\3ds Max 2019\\', 'ZES_ENABLE_SYSMAN': '1', 'HAXEPATH': 'C:\\HaxeToolkit\\haxe', 'PATHEXT': '.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW', 'TTOFF_LOGIN_TOKEN': 'dev', 'WINDIR': 'C:\\WINDOWS', 'PYCHARM COMMUNITY EDITION': 'C:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.2.2\\bin;', 'HOMEDRIVE': 'C:', 'SYSTEMDRIVE': 'C:', 'PUBLIC': 'C:\\Users\\Public', 'NUMBER_OF_PROCESSORS': '16', 'APPDATA': 'C:\\Users\\theph\\AppData\\Roaming', 'IDK2': 'C:\\Program Files\\Python38', 'COMMONPROGRAMW6432': 'C:\\Program Files\\Common Files', 'OS': 'Windows_NT', 'ADSK_CLM_WPAD_PROXY_CHECK': 'FALSE', 'IDK': 'C:\\Program Files\\Python38\\Scripts'}
:01-05-2023 23:24:27 TTOffLauncher: isTestServer: 0
:01-05-2023 23:24:27 TTOffLauncher: Game Server 127.0.0.1
:01-05-2023 23:24:27 TTOffLauncher: Starting Toontown Online... 
:01-05-2023 23:24:27 TTOffLauncher: foreground: Launcher now operating in foreground
useTTOffSpecificLogin=True
:01-05-2023 23:24:27 TTOffLauncher: maybeStartGame: starting game
:01-05-2023 23:24:27 TTOffLauncher: Starting Toontown Online...
:01-05-2023 23:24:27 TTOffLauncher: background: Launcher now operating in background
ToontownStart: Polling for game2 to finish...
ToontownStart: Game2 is finished.
ToontownStart: Starting the game.
ToontownStart: setting default font
:01-05-2023 23:24:28 TTOffLauncher: setting panda error code to 7
Known pipe types:
  wglGraphicsPipe
(all display modules loaded.)
:01-05-2023 23:24:28 ToonBase: Default graphics pipe is wglGraphicsPipe (OpenGL).
:01-05-2023 23:24:28 :display(warning): FrameBufferProperties available less than requested.
  requested: depth_bits=24 color_bits=3 red_bits=1 green_bits=1 blue_bits=1 alpha_bits=1 multisamples=70 back_buffers=1 force_hardware 
  got: depth_bits=24 color_bits=24 red_bits=8 green_bits=8 blue_bits=8 alpha_bits=8 accum_bits=64 multisamples=16 back_buffers=1 force_hardware 
:01-05-2023 23:24:28 ToonBase: Successfully opened window of type wglGraphicsWindow (OpenGL)
:01-05-2023 23:24:28 ShowBase: __dev__ == False
:01-05-2023 23:24:28 TTOffLauncher: setting panda error code to 11
:01-05-2023 23:24:28 TTOffLauncher: setting panda error code to 0
ToontownStart: Loading default gui sounds
ToontownStart: serverVersion:  tto-dev
:01-05-2023 23:24:28 Loader: starting bulk load of block 'init'
:01-05-2023 23:24:29 :nativenet(error): Buffered_DatagramConnection Constructor rbufsize = 4096000 wbufsize = 4096000 write_flush_point = 1400
:01-05-2023 23:24:29 OTPClientRepository: Using connect method 'http'
:01-05-2023 23:24:29 TTOffLauncher: getRegistry ('DEPLOYMENT', None)
:01-05-2023 23:24:29 TTOffLauncher: checking env
:01-05-2023 23:24:29 TTOffLauncher: getRegistry ('GAME_DEPLOYMENT', None)
:01-05-2023 23:24:29 TTOffLauncher: checking env
:01-05-2023 23:24:29 OTPClientRepository: reg_deployment=
:01-05-2023 23:24:29 OTPClientRepository: required-login auto.
:01-05-2023 23:24:29 OTPClientRepository: loginInterface: LoginTTOffAccount
:01-05-2023 23:24:29 TTOffLauncher: getNeedPwForSecretKey using ttoff-specific-login
:01-05-2023 23:24:29 Loader: At end of block 'init', expected 138, loaded 5, duration=1.2432178
:01-05-2023 23:24:29 OTPClientRepository: Connecting to gameserver directly (no proxy).
:01-05-2023 23:24:29 OTPClientRepository: Connecting to //127.0.0.1:7198 via HTTP interface.
:01-05-2023 23:24:29 TTOffLauncher: Stopping mini task manager.
:01-05-2023 23:24:29 OTPClientRepository: Successfully connected to //127.0.0.1:7198.
:01-05-2023 23:24:29 TTOffLauncher: Setting Disconnect Details normal
{402000001: <toontown.distributed.ToontownDistrict.ToontownDistrict instance at 0x0000000020CA0BC8>}
:01-05-2023 23:24:32 StateData: AvatarChooser.enter
:01-05-2023 23:24:32 DisplayOptions: settingsFullScreen = False, embedded = False width=800 height=600
:01-05-2023 23:24:32 StateData: calling self.displayOptions.restrictToEmbedded(False)
:01-05-2023 23:24:34 OTPClientRepository: ================
:01-05-2023 23:24:34 OTPClientRepository: Chose avatar id: 100000001
:01-05-2023 23:24:34 OTPClientRepository: Chose avatar name: Prof. Alvin Flippenhopper
:01-05-2023 23:24:34 OTPClientRepository: Chose avatar dna: ('dls', 'ss', 's', 'm', 15, 0, 15, 15, 1, 8, 1, 8, 1, 1)
:01-05-2023 23:24:34 OTPClientRepository: Chose avatar position: 1
:01-05-2023 23:24:34 OTPClientRepository: isPaid: 1
:01-05-2023 23:24:34 OTPClientRepository: freeTimeLeft: 0
:01-05-2023 23:24:34 OTPClientRepository: allowSecretChat: True
:01-05-2023 23:24:34 OTPClientRepository: ================
:01-05-2023 23:24:34 Loader: starting bulk load of block 'localAvatarPlayGame'
:01-05-2023 23:24:34 ToontownTimeManager(warning): error parsing date string=
local
:01-05-2023 23:24:35 NewsPageButtonManager(warning): Should not get here. NewsPageButtonManager.exitOff
:01-05-2023 23:24:35 :pnmimage:png(warning): iCCP: known incorrect sRGB profile
:01-05-2023 23:24:38 OTPClientRepository: Entering shard 402000001
:01-05-2023 23:24:38 StreetSign: Street sign url is https://ttoffline.com/images/
:01-05-2023 23:24:39 TimeManager: Clock sync: TimeManager.announceGenerate
:01-05-2023 23:24:39 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:24:39 InGameNewsMgr: latestIssue=2013-08-22 16:49:46.000003-07:00
cpu info: GenuineIntel|Intel(R) Core(TM) i9-9900K CPU @ 3.60GHz|591597|0|3.600,3.600|8,16 cpus
:01-05-2023 23:24:40 OTPClientRepository: No sync from TimeManager.
:01-05-2023 23:24:40 OTPClientRepository: gotTimeSync
:01-05-2023 23:24:40 Loader: At end of block 'localAvatarPlayGame', expected 400, loaded 19, duration=6.2647379
:01-05-2023 23:24:40 OTPClientRepository: Sending heartbeat mid-frame.
:01-05-2023 23:24:40 TimeManager: Clock sync roundtrip took 1123.766 ms
:01-05-2023 23:24:40 TimeManager: AI time delta is -0:01 from server delta
:01-05-2023 23:24:40 TimeManager: Local clock uncertainty +/- 0.562 s
:01-05-2023 23:24:40 PlayGame: QA-REGRESSION: NEIGHBORHOODS: Visit Cashbot HQ
:01-05-2023 23:24:40 Loader: starting bulk load of block 'hood'
:01-05-2023 23:24:41 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:24:41 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:24:41 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:24:41 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:24:41 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:24:41 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:24:42 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:24:42 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:24:42 :pnmimage:png(warning): Interlace handling should be turned on when using png_read_image
:01-05-2023 23:24:42 CashbotCogHQLoader: loadPlaceGeom: 12000
:01-05-2023 23:24:43 DistributedObject: DistributedSuitPlanner 402002974: generating
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/mb.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/mb.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:43 DistributedToon: calling LocalAvatar.startchat
:01-05-2023 23:24:43 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_cashbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:24:43 Loader: At end of block 'hood', expected 2, loaded 11, duration=2.6993193
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/mb.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/mb.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/mb.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/mb.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/bc.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/bc.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/mb.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/mb.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/bc.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/bc.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/mb.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/mb.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/mb.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/mb.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/nc.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/nc.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/nc.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/nc.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/mb.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/mb.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/tw.png
:01-05-2023 23:24:43 :gobj(error): Unable to find texture "phase_3/maps/hands/tw.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:24:44 Teleport(debug): teleportInDone: walk
:01-05-2023 23:24:44 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_cashbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:24:49 CashbotHQExterior: QA-REGRESSION: COGBATTLE: Enter Battle
Battle Event localtoon-battle-event-3
:01-05-2023 23:24:51 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/bc.png
:01-05-2023 23:24:51 :gobj(error): Unable to find texture "phase_3/maps/hands/bc.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:25:03 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/mb.png
:01-05-2023 23:25:03 :gobj(error): Unable to find texture "phase_3/maps/hands/mb.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:25:04 ToontownChatManager: QA-REGRESSION: CHAT: Speedchat Plus
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:25:07 TTOffMagicWordManager: Successfully skipped Battle movie!
:01-05-2023 23:25:08 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_cashbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:25:13 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/mb.png
:01-05-2023 23:25:13 :gobj(error): Unable to find texture "phase_3/maps/hands/mb.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:25:16 TTOffMagicWordManager: Run mode has been toggled.
:01-05-2023 23:25:20 TTOffMagicWordManager: Going ghost!
:01-05-2023 23:25:24 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/mb.png
:01-05-2023 23:25:24 :gobj(error): Unable to find texture "phase_3/maps/hands/mb.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:25:28 TTOffMagicWordManager: Abracadabra!
:01-05-2023 23:25:32 CashbotHQExterior: QA-REGRESSION: COGBATTLE: Enter Battle
Battle Event localtoon-battle-event-9
:01-05-2023 23:25:35 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/bc.png
:01-05-2023 23:25:35 :gobj(error): Unable to find texture "phase_3/maps/hands/bc.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:25:46 TTOffMagicWordManager: Successfully skipped Battle movie!
:01-05-2023 23:25:47 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/nc.png
:01-05-2023 23:25:47 :gobj(error): Unable to find texture "phase_3/maps/hands/nc.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:25:47 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_cashbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:25:49 CashbotHQExterior: QA-REGRESSION: COGBATTLE: Enter Battle
Battle Event localtoon-battle-event-15
:01-05-2023 23:25:58 ToontownChatManager: QA-REGRESSION: CHAT: Speedchat Plus
:01-05-2023 23:25:58 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/mb.png
:01-05-2023 23:25:58 :gobj(error): Unable to find texture "phase_3/maps/hands/mb.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:03 TTOffMagicWordManager: Successfully skipped Battle movie!
:01-05-2023 23:26:04 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_cashbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:26:08 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/nc.png
:01-05-2023 23:26:08 :gobj(error): Unable to find texture "phase_3/maps/hands/nc.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:09 ShtikerBook: QA-REGRESSION: SHTICKERBOOK: Open
OnscreenText.configure: mayChange == 0
OnscreenText.configure: mayChange == 0
OnscreenText.configure: mayChange == 0
OnscreenText.configure: mayChange == 0
:01-05-2023 23:26:10 ShtikerBook: QA-REGRESSION: SHTICKERBOOK: Browse tabs Map
:01-05-2023 23:26:11 ShtikerBook: QA-REGRESSION: SHTICKERBOOK: Close
:01-05-2023 23:26:16 CashbotHQExterior: Unloading Place (12053). Fsm in final
:01-05-2023 23:26:16 DistributedObject: DistributedSuitPlanner 402002974: disabling
:01-05-2023 23:26:16 PlayGame: QA-REGRESSION: NEIGHBORHOODS: Visit Lawbot HQ
:01-05-2023 23:26:16 Loader: starting bulk load of block 'hood'
:01-05-2023 23:26:17 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:26:17 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:26:17 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:26:17 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:26:17 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:26:17 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:26:17 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:26:17 TownBattleSuitPanel: Loading Suit Battle Panel!
:01-05-2023 23:26:17 LawbotCogHQLoader: loadPlaceGeom: 13000
:01-05-2023 23:26:17 ToonBase(warning): base.loadMusic is deprecated, use base.loader.loadMusic instead.
:01-05-2023 23:26:17 DistributedObject: DistributedSuitPlanner 402002984: generating
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:17 DistributedToon: calling LocalAvatar.startchat
:01-05-2023 23:26:17 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_lawbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:26:17 Loader: At end of block 'hood', expected 2, loaded 5, duration=1.3227888
:01-05-2023 23:26:17 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:17 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:17 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:17 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:17 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:17 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:17 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:17 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:17 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:17 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:17 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:17 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:17 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/ac.png
:01-05-2023 23:26:17 :gobj(error): Unable to find texture "phase_3/maps/hands/ac.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:17 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:17 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:17 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:17 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:18 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:18 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:18 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:18 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:18 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:18 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:19 Teleport(debug): teleportInDone: walk
:01-05-2023 23:26:19 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_lawbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:26:24 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:24 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:31 OTPClientRepository(warning): Don't know who friend 1 is.
:01-05-2023 23:26:34 LawbotHQExterior: QA-REGRESSION: COGBATTLE: Enter Battle
Battle Event localtoon-battle-event-21
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:46 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:46 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:48 TTOffMagicWordManager: Successfully skipped Battle movie!
:01-05-2023 23:26:56 TTOffMagicWordManager: Successfully skipped Battle movie!
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:57 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:26:57 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:26:58 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_lawbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:27:08 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:27:08 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:27:11 LawbotHQExterior: QA-REGRESSION: COGBATTLE: Enter Battle
Battle Event localtoon-battle-event-27
:01-05-2023 23:27:18 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:27:18 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:27:22 TTOffMagicWordManager: Successfully skipped Battle movie!
:01-05-2023 23:27:24 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_lawbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:27:29 LawbotHQExterior: QA-REGRESSION: COGBATTLE: Enter Battle
Battle Event localtoon-battle-event-33
:01-05-2023 23:27:40 TTOffMagicWordManager: Successfully skipped Battle movie!
:01-05-2023 23:27:40 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:27:40 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:27:40 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:27:40 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:27:41 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_lawbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:27:44 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_lawbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:27:45 ClassicFSM(warning): [DistributedSuit]: no transition exists from ToSky to WaitForBattle
:01-05-2023 23:27:45 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_lawbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:27:52 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:27:52 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:27:52 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/ac.png
:01-05-2023 23:27:52 :gobj(error): Unable to find texture "phase_3/maps/hands/ac.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:27:57 LawbotHQExterior: QA-REGRESSION: COGBATTLE: Enter Battle
Battle Event localtoon-battle-event-39
:01-05-2023 23:28:04 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:28:04 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:28:04 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:28:04 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:28:08 TTOffMagicWordManager: Successfully skipped Battle movie!
:01-05-2023 23:28:14 TTOffMagicWordManager: Successfully skipped Battle movie!
:01-05-2023 23:28:15 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:28:15 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:28:15 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:28:15 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:28:16 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_lawbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:28:27 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/dt.png
:01-05-2023 23:28:27 :gobj(error): Unable to find texture "phase_3/maps/hands/dt.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:28:28 LawbotHQExterior: QA-REGRESSION: COGBATTLE: Enter Battle
Battle Event localtoon-battle-event-45
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:28:38 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/sd.png
:01-05-2023 23:28:38 :gobj(error): Unable to find texture "phase_3/maps/hands/sd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:28:40 TTOffMagicWordManager: Successfully skipped Battle movie!
:01-05-2023 23:28:44 ToonBase: User closed main window.
:01-05-2023 23:28:44 TimeManager: Client disconnect reason 2.
:01-05-2023 23:28:44 DistributedObject: DistributedSuitPlanner 402002984: disabling
:01-05-2023 23:28:44 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_lawbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:01-05-2023 23:28:44 LawbotHQExterior: Unloading Place (13001). Fsm in final
:01-05-2023 23:28:44 OTPClientRepository(warning): unclean exit, abandoning shard
ignoring InputState-forward
:01-05-2023 23:28:44 OTPClientRepository: sending clientLogout
Assertion failed: _error_type != ET_not_found at line 592 of panda/src/pgraph/nodePath.cxx
Traceback (most recent call last):
  File "C:\Panda3D-1.11.0-x64\direct\showbase\EventManager.py", line 49, in eventLoopTask
    self.doEvents()
  File "C:\Panda3D-1.11.0-x64\direct\showbase\EventManager.py", line 43, in doEvents
    processFunc(dequeueFunc())
  File "C:\Panda3D-1.11.0-x64\direct\showbase\EventManager.py", line 99, in processEvent
    messenger.send(eventName, paramList)
  File "C:\Panda3D-1.11.0-x64\direct\showbase\Messenger.py", line 333, in send
    self.__dispatch(acceptorDict, event, sentArgs, foundWatch)
  File "C:\Panda3D-1.11.0-x64\direct\showbase\Messenger.py", line 418, in __dispatch
    result = method (*(extraArgs + sentArgs))
  File "toontown\toonbase\ToonBase.py", line 187, in windowEvent
    OTPBase.OTPBase.windowEvent(self, win)
  File "C:\Panda3D-1.11.0-x64\direct\showbase\ShowBase.py", line 2791, in windowEvent
    self.userExit()
  File "toontown\toonbase\ToonBase.py", line 422, in userExit
    self.cr.loginFSM.request('shutdown')
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 281, in request
    exitArgList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 227, in __transition
    self.__exitCurrent(exitArgList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 185, in __exitCurrent
    self.__currentState.exit(argList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\State.py", line 213, in exit
    self.__exitFunc(*argList)
  File "toontown\distributed\ToontownClientRepository.py", line 430, in exitPlayingGame
    base.localAvatar.deleteOrDelay()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 201, in deleteOrDelay
    self.disableAnnounceAndDelete()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 204, in disableAnnounceAndDelete
    self.disableAndAnnounce()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 230, in disableAndAnnounce
    self.disable()
  File "toontown\toon\LocalToon.py", line 256, in disable
    self.book.unload()
  File "toontown\shtiker\ShtikerBook.py", line 150, in unload
    page.unload()
  File "toontown\shtiker\SuitPage.py", line 240, in unload
    shadow.removeNode()
AssertionError: _error_type != ET_not_found at line 592 of panda/src/pgraph/nodePath.cxx
:01-05-2023 23:28:44 :task(error): Exception occurred in PythonTask eventManager
:01-05-2023 23:28:44 TTOffLauncher: setting panda error code to 12
:01-05-2023 23:28:44 TTOffLauncher(warning): Handling Python exception.
:01-05-2023 23:28:44 OTPClientRepository: Sent disconnect message to server
:01-05-2023 23:28:44 OTPClientRepository: Closing connection to server.
:01-05-2023 23:28:44 TTOffLauncher: Exception exit.

Traceback (most recent call last):
  File "otp\launcher\LauncherBase.py", line 1081, in mainLoop
    self._runTaskManager()
  File "otp\launcher\LauncherBase.py", line 1052, in _runTaskManager
    taskMgr.run()
  File "C:\Panda3D-1.11.0-x64\direct\task\Task.py", line 528, in run
    self.step()
  File "C:\Panda3D-1.11.0-x64\direct\task\Task.py", line 482, in step
    self.mgr.poll()
  File "C:\Panda3D-1.11.0-x64\direct\showbase\EventManager.py", line 49, in eventLoopTask
    self.doEvents()
  File "C:\Panda3D-1.11.0-x64\direct\showbase\EventManager.py", line 43, in doEvents
    processFunc(dequeueFunc())
  File "C:\Panda3D-1.11.0-x64\direct\showbase\EventManager.py", line 99, in processEvent
    messenger.send(eventName, paramList)
  File "C:\Panda3D-1.11.0-x64\direct\showbase\Messenger.py", line 333, in send
    self.__dispatch(acceptorDict, event, sentArgs, foundWatch)
  File "C:\Panda3D-1.11.0-x64\direct\showbase\Messenger.py", line 418, in __dispatch
    result = method (*(extraArgs + sentArgs))
  File "toontown\toonbase\ToonBase.py", line 187, in windowEvent
    OTPBase.OTPBase.windowEvent(self, win)
  File "C:\Panda3D-1.11.0-x64\direct\showbase\ShowBase.py", line 2791, in windowEvent
    self.userExit()
  File "toontown\toonbase\ToonBase.py", line 422, in userExit
    self.cr.loginFSM.request('shutdown')
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 281, in request
    exitArgList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 227, in __transition
    self.__exitCurrent(exitArgList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 185, in __exitCurrent
    self.__currentState.exit(argList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\State.py", line 213, in exit
    self.__exitFunc(*argList)
  File "toontown\distributed\ToontownClientRepository.py", line 430, in exitPlayingGame
    base.localAvatar.deleteOrDelay()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 201, in deleteOrDelay
    self.disableAnnounceAndDelete()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 204, in disableAnnounceAndDelete
    self.disableAndAnnounce()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 230, in disableAndAnnounce
    self.disable()
  File "toontown\toon\LocalToon.py", line 256, in disable
    self.book.unload()
  File "toontown\shtiker\ShtikerBook.py", line 150, in unload
    page.unload()
  File "toontown\shtiker\SuitPage.py", line 240, in unload
    shadow.removeNode()
AssertionError: _error_type != ET_not_found at line 592 of panda/src/pgraph/nodePath.cxx
